kyo-data/shared/src/test/scala/kyo/RecordTest.scala

```diff
@@ -297,7 +297,8 @@ class RecordTest extends Test:
             inline def stage[Name <: String, Value](field: Field[Name, Value]): Column[Value] =
                 Column[Value](field.name)(using summonInline[AsColumn[Value]])
 
-        "build record if all inlined" in {
+        "build record if all inlined" in pendingUntilFixed {
```

Comment:
@road21 this is still crashing. I've tried a few things without success. I'll open an issue to follow up.

Reply:
https://github.com/getkyo/kyo/pull/1187